rusk-recovery/src/state.rs

```diff
@@ -126,6 +126,9 @@ where
 
     info!("{} network state", theme.action("Storing"));
 
+    network.commit();
+    network.push();
```

Comment:
Looks like we need a method of `network` that will `commit` and `push`.

Reply:
That's exactly what I want to avoid in `rusk-vm`, however it's indeed what the `RuskState` does: it simplify that level of granularity. So, `finalize` is both `commit` and `push`.